### DISCUSSION TITLE
blog/format-nestjs-response

### DISCUSSION THREAD

**Discussion by giscus** [adarshaacharya/portfolio-giscus#3] [Announcements]
# blog/format-nestjs-response

Learn how to format Nest.js response and make standard response for every api response using Interceptors.

https://www.adarshaacharya.com.np/blog/format-nestjs-response

**Comment by mukezhz** [comment]
Nice implementation really informative 💡

**Reply by adarshaacharya** [reply]
Thanks!

**Comment by agustinustheo** [comment]
Nice, I am going to feature this in my NestJS content curation page if you don't mind!

**Reply by agustinustheo** [reply]
Here is my post: https://www.linkedin.com/feed/update/urn:li:activity:7191460455385604096

**Reply by adarshaacharya** [reply]
yes please ! thank you for sharing <3

**Comment by khayrullaev** [comment]
Great article but there are some considerations. You passed `TransformInterceptor` instead of `ResponseInterceptor` to the `app.useGlobalInterceptor`. And instead of _status_, using _success_ boolean would be more semantic.
